<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Minesweeper</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #3b404e;
font-family: Montserrat;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="topMargin">
     <number>40</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <property name="leftMargin">
       <number>30</number>
      </property>
      <property name="rightMargin">
       <number>30</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QPushButton" name="backBtn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
	background: #3b404e;
	border: transparent;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="iconSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Montserrat</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #ffffff;</string>
        </property>
        <property name="text">
         <string>〈Minesweeper〉</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
	border: none;
	color: #e5e5e7;
	background: transparent;
	border-bottom: 1px solid #575b67;
}

QLineEdit:focus {
	color: #dcdddf;
	border-bottom: 1px solid#eca61f;
}

QPushButton {
	color: #ffffff;
	background: #21242c;
	border: 10px solid #21242c;
    border-radius: 22px;
}

QPushButton:hover {
	background: #1a1c23;
	border: 10px solid  #1a1c23;
}</string>
      </property>
      <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0">
       <property name="leftMargin">
        <number>30</number>
       </property>
       <property name="rightMargin">
        <number>30</number>
       </property>
       <property name="horizontalSpacing">
        <number>20</number>
       </property>
       <item row="5" column="0" colspan="3">
        <widget class="QLabel" name="status">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">color: #c96174;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="3">
        <widget class="QLineEdit" name="passwordInput">
         <property name="font">
          <font>
           <family>Montserrat</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
         <property name="placeholderText">
          <string>Пароль</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="addImageBtn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	background: #3b404e;
	border: 2px solid #ffffff;
    border-radius: 28%;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="iconSize">
          <size>
           <width>50</width>
           <height>52</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="3">
        <widget class="QLineEdit" name="loginInput">
         <property name="font">
          <font>
           <family>Montserrat</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="placeholderText">
          <string>Логин</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="imagePlaceholder">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="3">
        <widget class="QPushButton" name="registerBtn">
         <property name="font">
          <font>
           <family>Montserrat</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Зарегистрироватья</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="3">
        <widget class="QLineEdit" name="nicknameInput">
         <property name="font">
          <font>
           <family>Montserrat</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="placeholderText">
          <string>Ник</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="3">
        <widget class="QLabel" name="lbl2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Montserrat</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: #ffffff;</string>
         </property>
         <property name="text">
          <string>Регистрация</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="lbl3">
         <property name="font">
          <font>
           <family>Montserrat</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: #7f828b</string>
         </property>
         <property name="text">
          <string>Фотография профиля</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
